doc/nrf/gs_installing.rst

```diff
@@ -316,38 +316,58 @@ To switch to the latest state of development, enter the following commands::
    Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
    This will bring the project repositories to the matching revision defined by the manifest file.
 
-Updating your existing clones to use west
-=========================================
+.. _repo_move:
 
-If you already cloned the |NCS| repositories in Git and want to continue using these clones instead of creating new ones, you can initialize west to use your clones.
-All branches, remotes, and other configuration in your repositories will be maintained.
+Pointing the repositories to the right remotes after they were moved
```

Comment:
According to Joel, we're not supposed to document this in the installation instructions, but it's only supposed to be added to the release notes. Please discuss with him.

Reply:
@jdstapes